### PR TITLE
Add support for Python 3.8. Drop support for Python 3.4 and 3.5.

### PR DESCRIPTION
**Pull request by icemac**
This is a prerequisite for #13.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def read(*rnames):
 ]
 
 setup(name='zope.session',
-      version='4.3.1.dev0',
+      version='4.4.dev0',
```
Comment (jamadden) [inline setup.py]:
```suggestion
      version='4.4.0.dev0',
```

File: CHANGES.rst [hunk 2]
```diff
@@ -2,10 +2,12 @@
  CHANGES
 =========
 
-4.3.1 (unreleased)
-==================
+4.4 (unreleased)
+================
```
Comment (jamadden) [inline CHANGES.rst]:
```suggestion
4.4.0 (unreleased)
==================
```

For consistency with all the previous releases.

**Review by jamadden — APPROVED** [review]
Looks pretty good to me.

File: CHANGES.rst (hunk 2 above)
Comment (icemac) [reply CHANGES.rst]: It seems my configuration of zest.releases is not so optimal for zopefoundation projects.

**Pull request merged**